source/verify-packages.txt

```diff
@@ -0,0 +1,244 @@
+            gpg --import atlas-cli.asc
+            
+         .. output::
+
+            gpg: key <key-value-short>: public key "Atlas CLI Release Signing Key <packaging@mongodb.com>" imported
+            gpg: Total number processed: 1
+            gpg:               imported: 1
+
+   .. step:: Verify the {+atlas-cli+} installation file.
+
+      Run the following command to verify the installation file:
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+
+            gpg --verify mongodb-atlas-cli_{+atlas-cli-version+}_linux_x86_64.tar.gz.sig mongodb-atlas-cli_{+atlas-cli-version+}_linux_x86_64.tar.gz
+            
+         .. output::
+
+            gpg: Signature made Thu Mar 14 08:25:00 2024 EDT
+            gpg:                using RSA key <key-value-long>
+            gpg: Good signature from "Atlas CLI Release Signing Key <packaging@mongodb.com>" [unknown]
+
+      If the package is properly signed, but you do not currently trust
+      the signing key, ``gpg`` also returns the following message :
+
+      .. code-block::
+
+         gpg: WARNING: This key is not certified with a trusted signature!
+         gpg:          There is no indication that the signature belongs to the owner.
+
+.. _verify-packages-windows:
+
+Verify Windows Packages
+-----------------------
+
+The following procedure verifies the {+atlas-cli+} package against its 
+SHA-256 key.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Download the {+atlas-cli+} installation file.
+
+      Download the {+atlas-cli+} ``.msi`` or ``.zip`` file from the 
+      `MongoDB Download Center
+      <https://www.mongodb.com/try/download/atlascli>`__ or 
+      `Github <https://github.com/mongodb/mongodb-atlas-cli/releases>`__.
+
+
+
+   .. step:: Save the public signature.
+
+      a. Open the ``checksums.txt`` file for the
+         release, which returns the
+         SHA256 key. For example, for version {+atlas-cli-version+},
+         download and open the following file: `https://github.com/mongodb/mongodb-atlas-cli/releases/download/atlascli%2Fv{+atlas-cli-version+}/checksums.txt <https://github.com/mongodb/mongodb-atlas-cli/releases/download/atlascli%2Fv{+atlas-cli-version+}/checksums.txt>`__.
+      #. Copy the text listed next to the file you downloaded. For 
+         example, if you downloaded ``mongodb-atlas-cli_{+atlas-cli-version+}_windows_x86_64.zip``, 
+         copy the text next to 
+         ``mongodb-atlas-cli_{+atlas-cli-version+}_windows_x86_64.zip``.
+         This is the SHA-256 key value.
+      #. Save the key value in a ``.txt`` file named ``atlas-cli-key``
+         in your Downloads folder.
+
+   .. step:: Compare the signature file to the {+atlas-cli+} installer hash.
+
+      Run the Powershell command to verify the package based on the
+      file you downloaded. 
+      
+      If you downloaded 
+      ``mongodb-atlas-cli_{+atlas-cli-version+}_windows_x86_64.zip``,
+      run the following command:
```

Comment:
Should we say copy and paste and replace the version variable?

Reply:
Since these docs are versioned, this command and example should reflect the version of the docs they're looking at, so I think this works as-is, but I might be misunderstanding what you're suggesting here. The only reason I call this out is that the command differs based on whether they downloaded the .zip or the .msi